compiler-core/src/build/package_compiler.rs

```diff
@@ -146,7 +153,7 @@ where
         };
 
         // Load the cached modules that have previously been compiled
-        for module in loaded.cached.into_iter() {
+        for module in loaded.cached.clone().into_iter() {
```

Comment:
It seems rather wasteful to clone all this information, especially since it's only going to be sometimes used, and it already stored elsewhere. Could instead we reference the copy that is stored in the existing_modules collection?

Reply:
I'm sorry, but I can't understand exactly what you mean. Do you mean reusing the modules stored in the `existing_modules` further down the line, as the cached modules were moved into it?

Reply:
I guess I could store the names of the cached modules, and retrieve those from the `existing_modules` collection further down the line, although that would require cloning anyway, right?

Reply:
Also should be fixed now